to-do-list/src/App.js

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from 'react';
+import './App.css';
+import TaskForm from './components/TaskForm';
+import TaskList from './components/TaskLists';
+
+function App() {
+
+  const [tasks, setTasks] = useState([]);
+  const [curDate, setCurDate] = useState(new Date())
+  const [status, setStatus] = useState("")
+
+  useEffect(() => {
+    const timer = setInterval(() => setCurDate(new Date()), 1000)
+    return function cleanup() {
+      clearInterval(timer)
+    }
+  })
+
+  const addTask = function (task, date, time) {
+    const newTask = [...tasks, { task, date: date +" "+ time, time: time, id: tasks.length }]
+    setTasks(newTask)
+  }
+
+  const completeTask = function(){
+    console.log("called completed task")
+    setStatus(status !== "complete" ? "complete" : "")
```

Comment:
Is this doing the function you want? What is the `status` state? When I click this button that calls this function, all of my todos are marked as complete instead of just the one.